proto/tabletmanagerdata.proto

```diff
@@ -268,6 +268,23 @@ message ExecuteFetchAsDbaResponse {
   query.QueryResult result = 1;
 }
 
+message ExecuteMultiFetchAsDbaRequest {
+  bytes sql = 1;
+  string db_name = 2;
+  uint64 max_rows = 3;
+  bool disable_binlogs = 4;
+  bool reload_schema = 5;
+  bool disable_foreign_key_checks = 6;
+}
+
+message ExecuteMultiFetchAsDbaResponse {
+  repeated query.QueryResult results = 1;
+
+  // Errors is a list of errors encountered while executing the queries.
+  // It is either nil/empty, or maps 1:1 to the results.
```

Comment:
Not sure I 100% understand this comment. Some questions: 

- What happens if there's say 4 queries and the 3rd one fails. Do we have `""`, `""`, `"the error"` and then a 4th element or not? 
- Would we try to execute the 4th query at all if there was a previous failure? 

Would it be `maps 1:1 to the results"?

Reply:
Yikes, that's a leftover from an initial iteration. It no longer exists, now removed.